server/src/main/kotlin/com/kamelia/hedera/core/Validation.kt

```diff
@@ -0,0 +1,141 @@
+package com.kamelia.hedera.core
+
+import com.kamelia.hedera.rest.core.DTO
+import io.ktor.http.*
+
+class ValidationException : Exception()
```

Comment:
```suggestion
class ValidationException : Exception() {

    override fun fillInStackTrace(): Throwable = this
}
```

We don't need to build the stacktrace as the exception is basically just a controlled goto. Easy perf gain.